### ISSUE TITLE
Add Support with Pongo2 or Jinja2

### ISSUE THREAD

**Issue by srPuebla** [lukasjarosch/go-docx#28]
It would really increidible add support with Pongo2 or Jinja2 to edit docx templates.

**Comment by lukasjarosch** [comment]
I agree 😄 Depending how this is implemented, could mean a lot of work though. 
Unfortunately I doubt that I'll be able to invest this time right now, 
I'll definetely revisit this if no-one wants to have a go 👍🏼

**Comment by TeCHiScy** [comment]
Jinja2 sounds great. However, considering the implemention level, it may be easier to leverage the capability of pongo2. I would like to take a look at this feature, if there is some spare time.

Edit: Seems #11, #24  can be also achieved by resolving this issue, if some kind of template engine is introduced.

**Comment by lukasjarosch** [comment]
You're right, implementing that would definetely be a huge benefit.
It's definetely an interesting challenge, as rendered templates still need to be valid WordprocessingML XML.
I'd love to see what you come up with, in case you find some time 🚀
